style.css

```diff
@@ -636,10 +590,10 @@ button:disabled {
 
 .feedback {
   display: inline-flex;
-  padding: 120px 24px;
+  padding-top: auto;
   flex-direction: column;
-  align-items: flex-start;
-  gap: 58px;
+  justify-content: center;
+  align-items: center;
 }
```

Comment:
- [ ] Good job with the contact section so far 👍👍 Kindly make sure that it looks almost identical to the given Figma template as much as possible as [suggested](https://github.com/allenkays/Portfolio-setup-and-mobile-first/pull/4#discussion_r1264575018). Please fix this as well 😃
- [ ] Also, I noticed that **_First name and Last name_** labels are present in the contact form. Previously only the label for **_Full name_** was present for the name field. Kindly fix this also ⛏💪 In the last review, the reviewer didn't asked for adding the two input fields for name as it is given in the project requirements to use only one input field for it. However, kindly fix the design in terms of padding/margin/space and alignment for getting a design close to the Figma template as requested earlier 💪

![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/106119105/32fdb6e3-6d2c-46ee-bb94-ef217b6b8300)